### ISSUE TITLE
Could this be an opportunity to rebrand ?

### ISSUE THREAD

**Issue by WebReflection** [nodejs/modules#69]
TL;DR what if we take this new extension approach as an opportunity to rebrand JavaScript as ECMAScript so that [cases like the following one](https://www.reddit.com/r/javascript/comments/8d0bg2/oracle_owns_javascript_so_apple_is_taking_down_my/) would never ever possibly happen in the future?

## .es VS .mjs

  * entry points / apps that are not module don't carry meaningless extensions with them
  * ECMAScript is the well known standard that defines JavaScript, more meaningful than `.es` to describe what's the content is highly improbable
  * people will start thinking, and talking, about ES instead of JS
  * the future would not hold copyrights

Everything else valid for `.mjs` would be valid for `.es` too.

Is this so insane, as a proposal ?

#### common counter argument

> That's Spanish and Spain thing ... 

The `.fi` extensions never had this issue.

There was the intent to buy an island to have `.js` domani ... here he have already a country that would give us that, without conflicting with its native language which is, indeed, not English (the language in which ECMAScript is defined)

**Comment by ljharb** [comment]
The modules group doesn’t seem like the right place to have this discussion, but tldr, apple’s policies notwithstanding, it’s highy unlikely that sun/oracle’s trademark is defensible, but nobody’s tested it in court that I’m aware of.

Separately, the name “ecmascript” was chosen specifically so nobody would ever want to use it, so i don’t think encouraging its use it is a good idea.

Whether the extension is .es or .mjs or whatever is a totally fine bikeshed to have here, of course :-) but I’d vote against .es for the reason given above.

**Comment by mAAdhaTTah** [comment]
@ljharb If you're still bikeshedding the extension, was `.m.js` ever considered?

**Comment by ljharb** [comment]
@mAAdhaTTah the extension is only the part after the last dot, so that’d just be “.js” - it was suggested and rejected for that reason.

**Comment by MylesBorins** [comment]
@WebReflection the shortform "js" has never been an issue. I don't believe the language is going to change it's name any time in the near future, as such this seems like a distraction.

@mAAdhaTTah it was discussed and it was decided that it was not something to move forward with.

At his point I don't believe the extension discussion is going to be revisited. We've already kicked off the process at other standards organizations to make the extension official and the ecosystem has already begun adoption.

I think we should close this issue
